sessions/GraphQL-100/102/client/src/components/OrdersPage.js

```diff
@@ -17,7 +17,7 @@ const OrdersPage = props => (
     <Query query={GET_ORDERS}>
       {({ loading, error, data }) => {
         if (loading) return <p>Loading...</p>;
-        if (error) return <p>Oops something went wrong 😵</p>;
+        if (error) return <p>Oops something went wrong</p>;
         console.log(data.getOrders);
```

Comment:
Is this meant to be here?

Reply:
I removed the emoji to resolve a warning message